data/Pakistan/Assembly/sources/idmap/official.csv

```diff
@@ -349,3 +347,4 @@ id,uuid
 1053,c03a2279-1684-479a-8165-c6098ebb1174
 1054,5ed669b9-d426-4b19-8298-53cc6598d9e9
 1056,f67e27f4-7419-425c-879b-c89672659464
+1057,47223c4f-9ca3-4d33-928a-258adee23f2d
```

Comment:
This seems to have slipped into an earlier commit. It's not really part of archiving off the vanishing members.

Reply:
Oops, I've tidied up that commit so it only includes archive related things now.